README.md

```diff
@@ -20,11 +22,30 @@ bench::easy1_1K                                293 (3,494 MB/s)             241
 ...
 ```
 
+If you want to compare multiple implementations that can run the same benchmark
+you need only reuse the same test names in differently named modules. Then your
+benchmark output will look like `implementation::test`, and you can instruct
+benchcmp to compare two implementations with:
+
+```
+$ cargo benchcmp dynamic-no-lazy-dfa dynamic benchoutput
```

Comment:
Is this right? Aren't `dynamic-no-lazy-dfa` and `dynamic` supposed to be names of implementations? And is putting `benchoutput` there going to result in emitting contents to `stdout` (as shown here) or to the file `benchoutput`?